tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.html

```diff
@@ -113,10 +118,14 @@
         _runToTag: Object,  // map<run: string, tags: string[]>
         _dataNotFound: Boolean,
 
+        _tagFilter: {
+          type: String,  // upward bound from paper-input
+          value: '.*',
```

Comment:
I think we should bind the search query to the url, on a per-dashboard basis.
I don't need for that to land in this PR.